### DISCUSSION TITLE
PK-Sim Biometric Tables

### DISCUSSION THREAD

**Discussion by jhunt8** [Open-Systems-Pharmacology/Forum#1495] [Questions & Problems]
Hello All, 

This discussion concerns the biometric tables of PK-Sim to generate individuals and populations. 

Part 1: How may I view the tables to ascertain the limits on individual characteristics? For example, it seems that it is not possible to create individuals with some combinations of biometric characteristics:
![image](https://user-images.githubusercontent.com/87446546/229581227-7de841cb-9eb8-4933-8475-db5ae5701814.png)
(a 30 yr old man 50 kg but 180 cm tall) 

Part 2 (more complex): I would like to create or modify spatial structures in MoBi and then carry these changes into modified PK-Sim biometric tables for population simulations. For example, if I wanted to create a virtual population of athletes with higher proportion of muscle (larger muscle organ). Or, if I added a tumor compartment and wanted to simulate a distribution of tumor sizes/tissue compositions. Ideally, I could also modify the PK-Sim algorithm that references these tables so as to generate virtual populations with a customized algorithm. 

Thank you very much!

**Comment by Yuri05** [comment]
Regarding you question 1: you can open the [PK-Sim database](https://github.com/Open-Systems-Pharmacology/PK-Sim/raw/develop/src/Db/PKSimDB.sqlite) with any SQLite viewer (e.g. you can use the online viewer https://sqliteviewer.app/ )
![grafik](https://user-images.githubusercontent.com/25061876/230427974-d934f6af-1b08-4222-9978-98346d914edd.png)

E.g. in the screenshot above you can see that the probabilty distribution for the 30yo male individual for the population NHANES White American is given by the lognormal distribution with Mean=80.35kg and GeoSD=1.196: 
![grafik](https://user-images.githubusercontent.com/25061876/230427780-d75b9fd8-5eb9-4cf8-9ed9-2a5272acdfb2.png)

However the problem is not the distribution of weight (or height) itself, but the values in context of the [Create Individual algorithm](https://github.com/Open-Systems-Pharmacology/OSPSuite.Documentation/wiki/Create-Individual-Algorithm), which tries to create the "most probable individuum" for the given combination of {Weight, Height} based on the probability distributions of all organs.

If e.g. the weight is extremely low or extremely high, this might fail.
For those cases, better Create Individual algorithms are under discussion (@aedginto maybe you can comment on this).

Regarding the 2nd question: you can create a population in PK-Sim first and export it to CSV. Then you could adjust the values in the table following your own algorithm (you can also add new columns to the table, e.g. if you have added a tumor tissue in MoBi, you can add columns for the tumor volume, blood flow etc.).
Then you can import this adjusted population back into PK-Sim and either use it to create new simulations in PK-Sim or combine it with imported MoBi models.

**Reply by Yuri05** [reply]
Please also check the discussion https://github.com/Open-Systems-Pharmacology/Forum/discussions/875 
and the docu https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/importing-exporting-project-data-models#importing-individual-and-population-simulation
